### PR TITLE
Feature/jwt encode

### PR DESCRIPTION
**Pull request by beatrizmonteiromendes**
## Why?
Create a JWT using method for representing claims securely between two parties.

## Changes
Add JWT method 

## How to test
n/a

## Preview
n/a

Issues
- https://github.com/acesso-io/squad-check-mobile/issues/284

### CODE REVIEW

File: AcessoBio.podspec [hunk 1]
```diff
@@ -125,11 +125,13 @@ Pod::Spec.new do |spec|
   # spec.library   = "iconv"
   # spec.libraries = "iconv", "xml2"
 
-    #spec.dependency ""
+    #spec.dependency  'JWT'
```
Comment (henrique-cocito) [inline AcessoBio.podspec:128]: Could you remove this commented line?

Comment (MatheusDomingos) [inline AcessoBio.podspec:128]: Why two dependencies? Do we need both?

File: AcessoBio/UnicoCheck.m [hunk 2]
```diff
@@ -488,30 +488,30 @@ - (void)openCameraSelfie:(id<AcessoBioSelfieDelegate>)delegate {
     
     if ([self verifyTarget]) {
                 
-        [self initFaceTec];
+//        [self initFaceTec];
```
Comment (henrique-cocito) [inline AcessoBio/UnicoCheck.m:491]: Could you remove this commented line?

**Review by henrique-cocito — CHANGES_REQUESTED** [review]

**Review by MatheusDomingos — CHANGES_REQUESTED** [review]

File: AcessoBio.podspec (hunk 1 above)
Comment (beatrizmonteiromendes) [reply AcessoBio.podspec:128]: done

File: AcessoBio/UnicoCheck.m (hunk 2 above)
Comment (beatrizmonteiromendes) [reply AcessoBio/UnicoCheck.m]: done

File: AcessoBio.podspec (hunk 1 above)
Comment (beatrizmonteiromendes) [reply AcessoBio.podspec]: done

**Review by MatheusDomingos — APPROVED** [review]

**Review by MatheusDomingos — CHANGES_REQUESTED** [review]
Remove dataSecret

**Review by henrique-cocito — APPROVED** [review]

**Review by MatheusDomingos — APPROVED** [review]

File: AcessoBio/UnicoCheck.m [hunk 3]
```diff
@@ -483,35 +483,32 @@ - (void)openCameraSelfie:(id<AcessoBioSelfieDelegate>)delegate {
...
     }
     
-    
     self.selfieDelegate = delegate;
     
     if ([self verifyTarget]) {
-                
-        [self initFaceTec];
 
-//        cView = [CameraFaceView new];
```
Comment (Diniz-Lucas) [inline AcessoBio/UnicoCheck.m:493]: Remove commented code.

**Pull request merged**